testing-plugin/src/main/kotlin/arrow/meta/plugin/testing/TestingLibraryWrapper.kt

```diff
@@ -10,11 +10,11 @@ import java.nio.file.Paths
 
 private const val DEFAULT_FILENAME = "Example.kt"
 
-internal data class CompilationResult(
+data class CompilationResult(
   val actualStatus: CompilationStatus,
   val log: String,
   val actualGeneratedFilePath: Path,
-  val classesDirectory: File
```

Comment:
There are different directories and we only use classes directory. I think outputDirectory could be confusing with the outputDirectory of the testing library.

Reply:
where is the outputDirectory of the testing library.  I couldnt find it.

Reply:
line 33

Reply:
So, you're saying that `classesDirectory` is the same as `outputDirectory`?

Reply:
They are different. That's the reason why I chose `classesDirectory`...